VoxelSniperForge/build.gradle

```diff
@@ -1,5 +1,5 @@
 plugins {
-    id 'net.minecraftforge.gradle' version '5.1.+'
+    id 'net.minecraftforge.gradle' version '[6.0,6.2)'
```

Comment:
Shouldn't this be one fixed version instead of a range? (I assume that is what this means..)

Reply:
This is straight from the upgrade guide. It should be correct

Reply:
For further questions, I refer you to the gif in the description

Reply:
ok